pkg/operation/seed/seed.go

```diff
@@ -239,7 +237,14 @@ func deployCertificates(ctx context.Context, seed *Seed, k8sSeedClient kubernete
 }
 
 // BootstrapCluster bootstraps a Seed cluster and deploys various required manifests.
-func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubernetes.Interface, seed *Seed, secrets map[string]*corev1.Secret, imageVector imagevector.ImageVector, componentImageVectors imagevector.ComponentImageVectors, conf *config.GardenletConfiguration) error {
+func BootstrapCluster(ctx context.Context,
+	k8sGardenClient, k8sSeedClient kubernetes.Interface,
+	seed *Seed, secrets map[string]*corev1.Secret,
```

Comment:
```suggestion
func BootstrapCluster(
	ctx context.Context,
	k8sGardenClient kubernetes.Interface,
	k8sSeedClient kubernetes.Interface,
	seed *Seed,
	secrets map[string]*corev1.Secret,
```

If you use multiple lines then let's do it consistently :)